============================= test session starts ==============================
platform darwin -- Python 3.12.8, pytest-8.4.2, pluggy-1.6.0 -- /Users/nate/github.com/prefecthq/prefect-mcp-server/.venv/bin/python3
cachedir: .pytest_cache
rootdir: /Users/nate/github.com/prefecthq/prefect-mcp-server
configfile: pyproject.toml
plugins: asyncio-1.2.0, anyio-4.11.0, logfire-4.9.0, sugar-1.1.1, rerunfailures-16.0.1
asyncio: mode=Mode.AUTO, debug=False, asyncio_default_fixture_loop_scope=None, asyncio_default_test_loop_scope=function
collecting ... collected 7 items

tests/test_server.py::test_server_has_expected_capabilities 17:19:20.039 | INFO    | prefect - Starting temporary server on http://127.0.0.1:8465
See https://docs.prefect.io/v3/concepts/server#how-to-guides for more information on running a dedicated Prefect server.
PASSED
tests/test_server.py::test_list_deployments_with_test_data PASSED
tests/test_server.py::test_run_deployment_by_name PASSED
tests/test_server.py::test_run_nonexistent_deployment_by_name PASSED
tests/test_server.py::test_identity_resource PASSED
tests/test_server.py::test_dashboard_resource PASSED
tests/test_server.py::test_read_events_tool [09/23/25 17:20:12] WARNING  Failed to get tools from server: tool_manager.py:86
                             'Prefect Docs Proxy', mounted                      
                             at: 'docs': Server error '500                      
                             Internal Server Error' for url                     
                             'https://docs.prefect.io/mcp'                      
                             For more information check:                        
                             https://developer.mozilla.org/en                   
                             -US/docs/Web/HTTP/Status/500                       
PASSED17:20:14.870 | INFO    | prefect - Stopping temporary server on http://127.0.0.1:8465


============================== 7 passed in 55.99s ==============================
